.github/workflows/integration.yml

```diff
@@ -199,8 +202,7 @@ jobs:
       run: ./gradlew lintFrontend
 
     - name: Build with Gradle
-      run: ./gradlew clean checkstyleMain checkstyleTest test systemTest coverage
-
+      run: Measure-Command { Start-Process -NoNewWindow -Wait ./gradlew "clean checkstyleAll test systemTest coverage" }
```

Comment:
I think it's okay we do not need to measure how long the process takes? Gradle already has a built-in timer and will report how long the build takes at the end.

Also, I realised you are using `gradlew`. Does this automatically use the `gradlew.bat` file?

Reply:
> Gradle already has a built-in timer

Ah I see, I put that in for consistency with the other tasks, but it does seem somewhat contrived.

> Does this automatically use the gradlew.bat file?

This is a great question. I was following some of [these](https://github.com/gradle/gradle-build-action/tree/main/.github/workflows) workflows which use `./gradlew` for all OSs - I'm not actually sure why this works. It's possible that powershell on these server images have some kind of bash support.

Reply:
I see, I think it might be because PowerShell supports running bash scripts.

Regarding the timing of the build commands, is it possible to remove it from the other workflows as well? So that we can make it more consistent for all 3 OSs